DisableVehicleNeonLights.md

```diff
@@ -0,0 +1,15 @@
+---
+ns: VEHICLE
+aliases: ["N_0x83f813570ff519de"]
```

Comment:
This alias type shouldn't have a N_ prefix.

Reply:
Fixed.